e2e_testing/xfail_sets.py

```diff
@@ -201,10 +186,87 @@
     'IsFloatingPointInt_False',
     'TorchPrimLoopForLikeModule_basic',
     'TorchPrimLoopWhileLikeModule_basic',
+
+    # backend never runs because of empty frame
```

Comment:
One of the main purpose of the XFAIL set is to serve as a living and up-to-date-by-construction set of actionable work items needed for the project to reach a desired state. For many of these, it's not clear what is actionable -- are these upstream bugs? (Please file them), or are they deliberately not working functionality upstream that won't be fixed?

Reply:
All of these have already been discussed and adjudicated here https://github.com/llvm/torch-mlir/pull/2040#issuecomment-1518303632.